client/keys/add_ledger_test.go

```diff
@@ -35,8 +35,11 @@ func Test_runAddCmdLedger(t *testing.T) {
 	assert.NoError(t, runAddCmd(cmd, []string{"keyname1"}))
 
 	// Now check that it has been stored properly
-	kb, err := NewKeyBaseFromHomeFlag()
-	assert.NoError(t, err)
+	kb := NewKeyringKeybase()
+	defer func() {
```

Comment:
This is unnecessary thanks to the above `defer kbCleanUp()`

Reply:
So these functions will CleanUp the keybase if using the OS secret store instead of the file backend which will be cleaned up by `kbCleanup()`

ie when running the tests locally on an OS with a secret store like a mac

Reply:
Oh right, that makes sense :+1: